### ISSUE TITLE
How to join my own server?

### ISSUE THREAD

**Issue by DillFrescott** [ssb-ngi-pointer/go-ssb-room#297]
Got everything set up and created an invite code. But it does nothing in the manyverse app on both desktop and iPhone. What am I doing wrong?

**Comment by staltz** [comment]
Have you created an invite link and clicked it?

**Comment by DillFrescott** [comment]
Yup. I've tried the button that opens the Manyverse app, that doesn't work, I tried the same on iPhone, doesn't work either.

I tried pasting the direct link into Manyverse and it says it was rejected for some reason. 

I tried just the token part of the link too.

Really confusing lol. 😭

**Comment by staltz** [comment]
Can you give (lots) more details? Like the version of go-ssb-room used, the version of Manyverse, your operating system and its version, and anything else "unusual" about your setup which may affect this issue.

**Comment by staltz** [comment]
@rmdes Perhaps what you're experiencing is a chmod permission problem on the SQLite file. Also, note that the Docker in this repo is not maintained by the maintainers, it was contributed by some community member, and we cannot guarantee it works flawlessly.

**Comment by rmdes** [comment]
No worries, maybe I can report feedback upstream, i'm going to look at the permission on the sqlite file

**Comment by rmdes** [comment]
> Got everything set up and created an invite code. But it does nothing in the manyverse app on both desktop and iPhone. What am I doing wrong?

Are you on a DEB based release install ?
if so I just went through the steps and when the package is installed, the best is to simply 
git clone this https://github.com/ssb-ngi-pointer/go-ssb-room 
you need to have Go properly installed (go env and go version should work)
and then 
follow the instruction from here https://github.com/ssb-ngi-pointer/go-ssb-room/blob/master/docs/deployment.md#first-admin-user

you will have to go inside the cmd/ directory and do "go build" inside the insert-user directory. 
once that is done simply copy paste the command from the link above with your own SSB profile
and you will have your first admin user. 

You can repeat that command with other @ ssb ID for friends that you want to put as admin or moderator.
